checker/src/org/checkerframework/checker/nullness/NullnessAnnotatedTypeFactory.java

```diff
@@ -299,6 +302,24 @@ protected AnnotatedTypeFormatter createAnnotatedTypeFormatter() {
                 super.methodFromUse(tree);
         AnnotatedExecutableType method = mfuPair.first;
 
+        // Change the type any uninferred type arguments to @NonNull @Initialized.
```

Comment:
Yes, this isn't working well and I also see lots of errors.
In the plume-lib log all cases seem to be with the un-inferred wildcard in the subtype and the supertype being a type parameter. Maybe that combination needs some extra love in the qualifier hierarchy?